lib/charms/layer/openstack.py

```diff
@@ -101,20 +113,35 @@ def _save_creds(creds_data):
         endpoint = attrs['auth-url']
         region = attrs['region']
 
-    if 'ca-certificates' in creds_data:
+    ca_cert = None
+    # seems like this might have changed at some point;
+    # newer controllers return the latter
+    trust_ca_keys = {'ca-certificates', 'cacertificates'}
+    if trust_ca_keys & creds_data.keys():
```

Comment:
I know this isn't going to be an expensive call, but past gaming self is sad that we do this twice instead of capturing it in a variable and using the variable. :)